src/node/extension/loadedScripts.ts

```diff
@@ -89,6 +86,14 @@ export class LoadedScriptsProvider implements TreeDataProvider<BaseTreeItem> {
 	getTreeItem(node: BaseTreeItem): TreeItem {
 		return node;
 	}
+
+	private async isSupportedDebugType(debugType: string | undefined, session: vscode.DebugSession): Promise<boolean> {
+		if (debugType === 'vslsShare') {
+			const debugSessionInfo = session ? await session.customRequest('debugSessionInfo', {}) : undefined;
```

Comment:
As per conversation with Rodrigo will add a Try-Catch around the customRequest in case this changes in future.